### PR TITLE
update index.html

### PR DESCRIPTION
**Pull request by techbelle**
make searchable and indexable by spiders; add meta data descriptors, change some content to use keywords more effectively.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -5,9 +5,12 @@
   <meta charset="utf-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1">
+<META NAME="ROBOTS" CONTENT="INDEX, FOLLOW">
```
Comment (Frick) [inline index.html]: I'd indent and lowercase the contents of this line to follow suit with what's already here already.

File: index.html [hunk 2]
```diff
@@ -5,9 +5,12 @@
   <meta charset="utf-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1">
+<META NAME="ROBOTS" CONTENT="INDEX, FOLLOW">
+	
   <!-- The above 3 meta tags *must* come first in the head; any other head content must come *after* these tags -->
```
Comment (Frick) [inline index.html]: I'd remove the empty line above this and move the comment up as it still stands. The `robots` line can come after.

File: index.html [hunk 3]
```diff
@@ -5,9 +5,12 @@
   <meta charset="utf-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1">
+<META NAME="ROBOTS" CONTENT="INDEX, FOLLOW">
+	
   <!-- The above 3 meta tags *must* come first in the head; any other head content must come *after* these tags -->
 
   <title>Arizona Med Services</title>
+<meta name="description" content="Arizona Medical Services provides state of the art, in office testing for the RPP Respiratory Pathogen Panel reducing costs and complexity for urgent care providers">
```
Comment (Frick) [inline index.html]: Indent

File: index.html [hunk 4]
```diff
@@ -144,13 +147,13 @@ <h2 class="title">About Us</h2>
         <!-- about -->
         <div class="col-md-10" id="about">
           <div class="about" id="about">
-            <p>We built our firm because we understand the ever changing landscape and complex needs that impact today's
-              providers. That’s why we have put together a unique team of highly trained and knowledgeable consultants to
+            <p>We understand the complex, evolving landscape impact today's
+              medical providers. We've built a unique team of highly trained and knowledgeable consultants to
```
Comment (Frick) [inline index.html]: The first sentence no longer makes sense to me. Maybe the below?
```
We understand the complex, evolving landscape that impacts today's medical providers.
```

File: index.html [hunk 5]
```diff
@@ -144,13 +147,13 @@ <h2 class="title">About Us</h2>
...
-              providers. That’s why we have put together a unique team of highly trained and knowledgeable consultants to
+            <p>We understand the complex, evolving landscape impact today's
+              medical providers. We've built a unique team of highly trained and knowledgeable consultants to
               help our clients capture lost revenue and stay compliant.</p>
-            <p>Our medical consulting focuses on a fundamental Process analysis of your existing structure. We make educated
-              and direct recommendations in processes and products, that streamline your operations and allow your facility
-              to capture more revenue. Most of our clients realized an overall increase in profitability of 25 to 50 percent
-              based on our very easily integrated recommendations after 60 days of implementation.</p>
+            <p>Our medical consulting focuses on a process analysis of your existing structure, data, and staff. We make educated
+              and direct recommendations in processes and products, that will streamline your operations and allow your facility
```
Comment (Frick) [inline index.html]: Remove the comma after `products`.

File: index.html [hunk 6]
```diff
@@ -248,9 +251,8 @@ <h2 class="title">Lab Services</h2>
             <i class="fa fa-universal-access"></i>
             <h3>Genetic Testing</h3>
             <p>Have the ability to move beyond trial and error. Genetic testing is the key to help providers bridge the gap
-              between diagnosis and prognosis. Now providers for the first time in history can map the genetics of a patient
-              and make a true comprehensive health and wellness plan. We offer many in network and out of network genetic
-              testing recommendations.
+              between diagnosis and prognosis. For the first time in history, providers can map the genetics of a patient
+              and make a true comprehensive health and wellness plan. Be part of this revolution in healthcare, proactively anticipate your patient's needs.
```
Comment (Frick) [inline index.html]:
```
Be part of this revolution in healthcare, proactively anticipate your patient's needs.
```
just doesn't read well to me. I'd think either of the below might be better.
```
Be part of this revolution in healthcare by proactively anticipating your patient's needs.
```
```
Be part of this revolution in healthcare. Proactively anticipate your patient's needs.
```

File: index.html [hunk 7]
```diff
@@ -263,7 +265,7 @@ <h3>Genetic Testing</h3>
           <div class="service">
             <i class="fa fa-code-fork"></i>
             <h3>Molecular Testing</h3>
-            <p>DNA testing and the subsequent mapping and reporting of pathogens and patient profiles can give unparalleled
+            <p>Alongside DNA testing, Arizona Medical Services can help your office deliver fast, convenient testing for specific pathogens and sensitivities - giving
```
Comment (Frick) [inline index.html:267]: Honestly... I kinda prefer the original statement. Perhaps just prefix the original with `Arizona Medical Service's `?

File: index.html [hunk 8]
```diff
@@ -307,19 +309,19 @@ <h3>Supplies</h3>
...
         </div>
 
         <div class="col-md-6 col-sm-6">
           <div class="service">
             <i class="fa fa-tasks"></i>
             <h3>Technology Interfacing Platforms</h3>
-            <p>We offer recommendations for faster and more user friendly data collection software that can help practitioners
-              make faster and more personalized treatment available to there patients. We also have the ability to custom
-              tailor programs that can work in tandem with your existing platforms.</p>
+            <p>User friendly, fast, thorough, secure data for doctors and nurses. Easy integrations. Training for your staff.
```
Comment (Frick) [inline index.html:321]: I'm not sure I like the old or new text. The old is a little long-winded (and uses `faster` twice), the new is a little harsh and sounds more like it should be a bulleted list.

**Review by Frick — CHANGES_REQUESTED** [review]
A few changes I'd definitely like to see, but some of the included is just discussion.

**Review by techbelle — COMMENTED** [review]
ready for review again

**Review by Frick — APPROVED** [review]
:man_shrugging:

**Comment by techbelle** [comment]
@bizarrowill can you please merge this pull request when you get a chance

**Pull request merged**